components/uuf-core/src/main/java/org/wso2/carbon/uuf/api/auth/SessionHandler.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.uuf.api.auth;
+
+import org.wso2.carbon.uuf.spi.HttpRequest;
+import org.wso2.carbon.uuf.spi.HttpResponse;
+import org.wso2.carbon.uuf.spi.auth.User;
+
+import java.util.Optional;
+
+/**
+ * Do operations related to session instances.
```

Comment:
javadocs meant to be informative. javadocs in this class just states the obvious and doesn't give any meaningful information. Refer http://blog.joda.org/2012/11/javadoc-coding-standards.html